chaoscenter/authentication/api/handlers/rest/project_handler.go

```diff
@@ -569,6 +592,20 @@ func LeaveProject(service services.ApplicationService) gin.HandlerFunc {
 			return
 		}
 
+		if member.Role == nil || *member.Role == entities.RoleOwner {
```

Comment:
The check should be instead
```suggestion
		if member.Role != nil && *member.Role == entities.RoleOwner {
```

Reply:
done